guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collection.adoc

```diff
@@ -97,6 +98,24 @@ If you are authenticating with a Kerberos ticket, map the `server_url` parameter
       use_gssapi: true
   tasks:
 ----
++
+If you need to use an HTTP proxy to reach the {Project} API, set the `https_proxy` environment variable:
```

Comment:
This doesn't look like an equivalent to the username/PAT/Kerberos step. In other words, this is something users would set _in addition_ to choosing username/PAT/Kerberos, isn't it? How about adding it in a separate substep?

```suggestion
.. If you need to use an HTTP proxy to reach the {Project} API, set the `https_proxy` environment variable:
```

Reply:
Yeah, it becomes a matrix. Ack!